### ISSUE TITLE
garmin formatter temporary files are not cleaned up

### ISSUE THREAD

**Issue by GoogleCodeExporter** [opencaching/okapi#246]
```
There are garminNNNN.zip files in the oc.de var/okapi directory which are as 
old as three weeks. Looks like the cleanup via register_shutdown_function() 
does not work.

I set restrict-view-commit as this can be easily exploited to fill up the whole 
disk space.
```

Original issue reported on code.google.com by `following09` on 11 May 2013 at 12:09

**Comment by following5** [comment]
@deg-pl 
Could you have a look in your $dynbasepath directory, if there are lots of old garmin*.zip files like in the [list above](https://github.com/opencaching/okapi/issues/246#issuecomment-122676069)?

**Comment by kojoty** [comment]
hi @following5, 
I don't know much about this issue but I can't see any garmin*.zip file at OCPL server...
There are only some number of files okapi_filecache* from last two days.

**Comment by following5** [comment]
Thanks. After some deep look into old OKAPI code, I now realized that his problem has been fixed by fbb930df in October 2014. :-)
